addon.py

```
@@ -1373,6 +1387,311 @@ def import_generated_asset_fal_ai(self, request_id: str, name: str):
+                return {
+                    "error": f"API request failed with status {response.status_code}: {response}"
+                }
+        except Exception as e:
+            return {"error": str(e)}
+
+    def poll_hunyuan_job_status(self, *args, **kwargs):
+        return self.poll_hunyuan_job_status_ai(*args, **kwargs)
+    
+    def poll_hunyuan_job_status_ai(self, job_id: str):
+        """Call the job status API to get the job status"""
+        print(job_id); 
```

Ruff (0.11.9)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)